content/hardware/04.pro/boards/portenta-c33/tutorials/user-manual/content.md

```diff
@@ -615,22 +615,32 @@ The Portenta C33 supports I<sup>2</sup>C communication, which allows data transm
 |       `39`/`SDA2`       |          `P302`         |
 |       `40`/`SCL2`       |          `P301`         |
 
-Please, refer to the [board pinout section](#pinout) of the user manual to find them on the board. The I<sup>2</sup>C pins are also available through the onboard ESLOV connector of the Portenta C33.
+To locate these pins on the board, please refer to the [board pinout section](#pinout) of the user manual. The `SDA0` and `SCL0` pins are also available through the onboard ESLOV connector of the Portenta C33.
+
+**Note on I2C resources**: System integrators can use the Portenta C33's High-Density connectors to expand the signals of the board to a custom-designed daughter board or carrier. Below is the I2C pins mapping on the board's High-Density connectors and shared resources:
+
+| **HD Connector** | **Interface** | **Pins** | **Status** | **Shared Peripherals** |
+|:----------------:|:-------------:|:--------:|:----------:|:----------------------:|
+|        J1        |     `I2C1`    |   43-45  |    Free    |            -           |
+|        J2        |     `I2C0`    |   44-46  |    Free    |            -           |
+|        J2        |     `I2C2`    |   45-47  |    Free    |            -           |
+
+***Status column indicates the current status of the pins. "Free" means the pins are not in use by another resource or peripheral of the board and are available for usage, while "Shared" means the pins are used by one or several resources or peripherals of the board.***

```

Comment:
```suggestion
To locate these pins on the board, please refer to the [board pinout section](#pinout) of the user manual. The `SDA0` and `SCL0` pins are also available through the onboard ESLOV connector of the Portenta C33.

***Note on I2C resources: System integrators can use the Portenta C33's High-Density connectors to expand the signals of the board to a custom-designed daughter board or carrier. Below is the I2C pins mapping on the board's High-Density connectors and shared resources:***

| **HD Connector** | **Interface** | **Pins** | **Status** | **Shared Peripherals** |
|:----------------:|:-------------:|:--------:|:----------:|:----------------------:|
|        J1        |     `I2C1`    |   43-45  |    Free    |            -           |
|        J2        |     `I2C0`    |   44-46  |    Free    |            -           |
|        J2        |     `I2C2`    |   45-47  |    Free    |            -           |

The status column indicates the current status of the pins. "Free" means the pins are not in use by another resource or peripheral of the board and are available for usage, while "Shared" means the pins are used by one or several resources or peripherals of the board.
```

Changes are for:
* The paparagrapth below can be considered as normal text/normal explanation. We are just explaining how the table works.
This way we can create a warning block on top for the system integrators